app/javascript/packages/document-capture/components/file-input.tsx

```diff
@@ -363,7 +375,7 @@ function FileInput(props: FileInputProps, ref: ForwardedRef<any>) {
           {!value && !isValuePending && (
             <div className="usa-file-input__instructions" aria-hidden="true">
               {bannerText && <strong className="usa-file-input__banner-text">{bannerText}</strong>}
-              {isMobile && bannerText ? null : (
+              {shouldShowHint() ? null : (
```

Comment:
The logic is a little more complex because this is also within a condition a few lines above `!value && !isValuePending` which controls whether the "inner" hint is rendered.